tests/zustand/atomWithStore.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { fireEvent, render, act } from '@testing-library/react'
+import create from 'zustand/vanilla'
+import { Provider, useAtom } from '../../src/index'
+import { atomWithStore } from '../../src/zustand'
+
+it('count state', async () => {
```

Comment:
Aren't there missing some tests, where Zustand store state is updated outside React and then you see those changes in the component?

Reply:
That's what I did in L39. (I'd admit the test is minimal.)